doc/ref/language.xml

```diff
@@ -1822,23 +1822,55 @@ Also see Chapter&nbsp;<Ref Chap="Functions"/>.
 <P/>
 <Index Subkey="definition by arrow notation">functions</Index>
 <Index>arrow notation for functions</Index>
-<C><A>arg-ident</A> -> <A>expr</A></C>
+<C>{ <A>arg-list</A> } -> <A>expr</A></C>
```

Comment:
So the old syntax is not documented anymore, is it? But there are still examples for it -- and I think we should still document it. Perhaps like this?
"If the argument list consists of a precisely a single argument, you can also omit the braces."